translations/map_elements.tex/es.po

```diff
@@ -122,10 +122,10 @@ msgstr ""
 "\\includegraphics[width=\\linewidth]{\\images/maptiles.png}\n"
 "\\begin{itemize}\n"
 "  \\footnotesize\n"
-"  \\item[\\textbf{1.}] Losetas Iniciales: I\n"
-"  \\item[\\textbf{2.}] Losetas Cercanas: II-II\n"
-"  \\item[\\textbf{3.}] Losetas Lejanas: IV-V\n"
-"  \\item[\\textbf{4.}] Losetas Centrales: VI-VII\n"
+"  \\item[\\textbf{1.}] Losetas Iniciales (S#): I\n"
+"  \\item[\\textbf{2.}] Losetas Cercanas (N#): II-II\n"
+"  \\item[\\textbf{3.}] Losetas Lejanas (F#): IV-V\n"
+"  \\item[\\textbf{4.}] Losetas Centrales (C#): VI-VII\n"
```

Comment:
```suggestion
"  \\item[\\textbf{1.}] Losetas Iniciales: I\n"
"  \\item[\\textbf{2.}] Losetas Cercanas: II-II\n"
"  \\item[\\textbf{3.}] Losetas Lejanas: IV-V\n"
"  \\item[\\textbf{4.}] Losetas Centrales: VI-VII\n"
```
Doesn't compile because of this.